cedar-policy/src/api/err.rs

```diff
@@ -213,7 +213,11 @@ pub mod human_schema_errors {
     #[derive(Debug, Error, Diagnostic)]
     #[error(transparent)]
     #[diagnostic(transparent)]
-    pub struct ParseError(#[from] pub(super) cedar_policy_validator::HumanSyntaxParseError);
+    pub struct ParseError {
+        /// The errors reported by the parser
+        #[from]
+        pub errors: cedar_policy_validator::HumanSyntaxParseError,
```

Comment:
If we're making the inner errors a public error type, do we still want this `ParseError` wrapper?

If we do, it might be more consistent with our other errors to make it `pub struct ParseErrors(...)` instead of a record struct.